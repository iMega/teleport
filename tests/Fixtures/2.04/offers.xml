<?xml version="1.0" encoding="UTF-8"?>
<КоммерческаяИнформация ВерсияСхемы="2.04" ДатаФормирования="2014-08-01T11:46:07">
	<ПакетПредложений СодержитТолькоИзменения="false">
		<Ид>d35a8ba6-d6ff-11dc-bb41-001a4b58ee99#</Ид>
		<Наименование>Пакет предложений</Наименование>
		<ИдКаталога>d35a8ba6-d6ff-11dc-bb41-001a4b58ee99</ИдКаталога>
		<ИдКлассификатора>d35a8ba6-d6ff-11dc-bb41-001a4b58ee99</ИдКлассификатора>
		<Владелец>
			<Ид>0a344413-48e9-11e2-a9c9-00259079b159</Ид>
			<Наименование>Резак каждому</Наименование>
			<ОфициальноеНаименование>ООО "Резак каждому"</ОфициальноеНаименование>
			<ИНН>780000000</ИНН>
			<РасчетныеСчета>
				<РасчетныйСчет/>
			</РасчетныеСчета>
		</Владелец>
		<ТипыЦен>
			<ТипЦены>
				<Ид>2e6a724e-0cc1-11de-8f0f-001e8c60f11c</Ид>
				<Наименование>Рублевая 4</Наименование>
				<Валюта>BYR</Валюта>
				<Налог>
					<Наименование>НДС</Наименование>
					<УчтеноВСумме>false</УчтеноВСумме>
				</Налог>
			</ТипЦены>
		</ТипыЦен>
		<Предложения>
			<Предложение>
				<Ид>348fceb0-219f-11e2-8401-001e8c60f11c#18b18a6d-66c6-11e2-8671-00259079b159</Ид>
				<Наименование>Привод Blue-Ray DVD-RW Liteon iHBS112-115</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">шт</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление>628 200 BYR за шт</Представление>
						<ИдТипаЦены>2e6a724e-0cc1-11de-8f0f-001e8c60f11c</ИдТипаЦены>
						<ЦенаЗаЕдиницу>628200</ЦенаЗаЕдиницу>
						<Валюта>BYR</Валюта>
						<Единица>шт</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>1</Количество>
			</Предложение>
			<Предложение>
				<Ид>1a4ec9af-1192-11e4-8372-00259079b159#1a4ec9b2-1192-11e4-8372-00259079b159</Ид>
				<Наименование>Привод DVD+/-RW LiteON IHAS124-14, 22x SuperMulti, SATA, Черный</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">шт</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление>147 000 BYR за шт</Представление>
						<ИдТипаЦены>2e6a724e-0cc1-11de-8f0f-001e8c60f11c</ИдТипаЦены>
						<ЦенаЗаЕдиницу>147000</ЦенаЗаЕдиницу>
						<Валюта>BYR</Валюта>
						<Единица>шт</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>2</Количество>
			</Предложение>
			<Предложение>
				<Ид>1a4ec9af-1192-11e4-8372-00259079b159#8174ffbb-17c0-11e4-8372-00259079b159</Ид>
				<Наименование>Привод DVD+/-RW LiteON IHAS124-14, 22x SuperMulti, SATA, Черный</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">шт</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление>144 800 BYR за шт</Представление>
						<ИдТипаЦены>2e6a724e-0cc1-11de-8f0f-001e8c60f11c</ИдТипаЦены>
						<ЦенаЗаЕдиницу>144800</ЦенаЗаЕдиницу>
						<Валюта>BYR</Валюта>
						<Единица>шт</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>1266</Количество>
			</Предложение>
			<Предложение>
				<Ид>b883f4ef-d3f8-11e2-b716-00259079b159#6740885c-0b58-11e4-99b3-00259079b159</Ид>
				<Наименование>Привод DVD+/-RW Samsung SH-224DB, 24x SuperMulti, SATA, Черный</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">шт</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление>147 000 BYR за шт</Представление>
						<ИдТипаЦены>2e6a724e-0cc1-11de-8f0f-001e8c60f11c</ИдТипаЦены>
						<ЦенаЗаЕдиницу>147000</ЦенаЗаЕдиницу>
						<Валюта>BYR</Валюта>
						<Единица>шт</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>1</Количество>
			</Предложение>
		</Предложения>
	</ПакетПредложений>
</КоммерческаяИнформация>