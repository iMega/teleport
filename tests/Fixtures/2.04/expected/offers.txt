{"entityType":40,"changes":true}
{"entityType":50,"guid":"348fceb0-219f-11e2-8401-001e8c60f11c#18b18a6d-66c6-11e2-8671-00259079b159","prod_guid":"348fceb0-219f-11e2-8401-001e8c60f11c","barcode":"","title":"Привод Blue-Ray DVD-RW Liteon iHBS112-115","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"1","postType":"product_variation"}
{"entityType":70,"offer_guid":"348fceb0-219f-11e2-8401-001e8c60f11c#18b18a6d-66c6-11e2-8671-00259079b159","title":"628 200 BYR за шт","price":"628200","currency":"BYR","unit":"шт","ratio":"1","type_guid":"2e6a724e-0cc1-11de-8f0f-001e8c60f11c"}
{"entityType":50,"guid":"1a4ec9af-1192-11e4-8372-00259079b159#1a4ec9b2-1192-11e4-8372-00259079b159","prod_guid":"1a4ec9af-1192-11e4-8372-00259079b159","barcode":"","title":"Привод DVD+\/-RW LiteON IHAS124-14, 22x SuperMulti, SATA, Черный","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"2","postType":"product_variation"}
{"entityType":70,"offer_guid":"1a4ec9af-1192-11e4-8372-00259079b159#1a4ec9b2-1192-11e4-8372-00259079b159","title":"147 000 BYR за шт","price":"147000","currency":"BYR","unit":"шт","ratio":"1","type_guid":"2e6a724e-0cc1-11de-8f0f-001e8c60f11c"}
{"entityType":50,"guid":"1a4ec9af-1192-11e4-8372-00259079b159#8174ffbb-17c0-11e4-8372-00259079b159","prod_guid":"1a4ec9af-1192-11e4-8372-00259079b159","barcode":"","title":"Привод DVD+\/-RW LiteON IHAS124-14, 22x SuperMulti, SATA, Черный","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"1266","postType":"product_variation"}
{"entityType":70,"offer_guid":"1a4ec9af-1192-11e4-8372-00259079b159#8174ffbb-17c0-11e4-8372-00259079b159","title":"144 800 BYR за шт","price":"144800","currency":"BYR","unit":"шт","ratio":"1","type_guid":"2e6a724e-0cc1-11de-8f0f-001e8c60f11c"}
{"entityType":50,"guid":"b883f4ef-d3f8-11e2-b716-00259079b159#6740885c-0b58-11e4-99b3-00259079b159","prod_guid":"b883f4ef-d3f8-11e2-b716-00259079b159","barcode":"","title":"Привод DVD+\/-RW Samsung SH-224DB, 24x SuperMulti, SATA, Черный","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"1","postType":"product_variation"}
{"entityType":70,"offer_guid":"b883f4ef-d3f8-11e2-b716-00259079b159#6740885c-0b58-11e4-99b3-00259079b159","title":"147 000 BYR за шт","price":"147000","currency":"BYR","unit":"шт","ratio":"1","type_guid":"2e6a724e-0cc1-11de-8f0f-001e8c60f11c"}
